pkg/core/transaction/oracle.go

```diff
@@ -22,11 +22,14 @@ const MaxOracleResultSize = math.MaxUint16
 
 // Enumeration of possible oracle response types.
 const (
-	Success   OracleResponseCode = 0x00
-	NotFound  OracleResponseCode = 0x10
-	Timeout   OracleResponseCode = 0x12
-	Forbidden OracleResponseCode = 0x14
-	Error     OracleResponseCode = 0xff
+	Success              OracleResponseCode = 0x00
+	ConsensusUnreachable OracleResponseCode = 0x10
+	NotFound             OracleResponseCode = 0x12
+	Timeout              OracleResponseCode = 0x14
+	Forbidden            OracleResponseCode = 0x16
+	ResponseTooLarge     OracleResponseCode = 0x18
```

Comment:
But how about `IsValid()`? It is used for decoding, we need to add them to `IsValid` too.

Reply:
Fixed.